jobs/gtfs-schedule-validator/README.md

```diff
@@ -2,10 +2,24 @@
 
 This image exists to faciliate a pod operator executing a Java JAR which validates
 GTFS Schedule zipfiles and outputs the resulting notices (i.e. violations). The
-JAR in this folder is sourced from https://github.com/MobilityData/gtfs-validator/
-and is currently v2.0.0 of that project.
+JARs in this folder are sourced from https://github.com/MobilityData/gtfs-validator/,
+and our Python code automatically selects which version is relevant for an extract.
 
 Note that unlike `gtfs-rt-parser`, this code does NOT include any parsing/unzipping
 of the underlying data. The RT code was written prior to substantial changes to
 `calitp-py` and therefore bundles together shared functionality that will eventually
 live in `calitp-py`.
+
+See [this PR](https://github.com/cal-itp/data-infra/pull/2893) for an example of how
+to integrate new versions of the underlying MD-stewarded validator into our validation
+process - specifically, changes to [`gtfs_schedule_validator_hourly.py`](https://github.com/cal-itp/data-infra/pull/2893/files#diff-f970d61c9ce49b3899e7b309610f43fca51d29eaae4d5db192208540bc4da702) and
+[the `Dockerfile`](https://github.com/cal-itp/data-infra/pull/2893/files#diff-de54f054016ca4e44544d4081d2798a21aa43790c3faea3967f6f5b255529f1a), and the
+addition of [a new JAR file](https://github.com/cal-itp/data-infra/pull/2893/files#diff-628dcc44a01286366e42ab8fd440d213d764d8a76f41f268e378a14628e8447c)
+corresponding to the new validator version being referenced (taken from the Assets
```

Comment:
nit: I don't know if the commit-level links are that useful

Reply:
I understand where you're coming from, but I think I'm gonna leave these as-is. The reason I'm including these file-changes-on-PR-level links is because we wanted to use this PR as an example of a validator upgrade, but the PR also includes some things not directly related to the upgrade (like getting rid of the non-hourly validation script). So to avoid confusion for a reader down the road, I decided to point out these specific changes as the ones relevant to the upgrade.

I also considered just using an older upgrade PR as an example, but the underlying code has changed so much since then (the previous upgrade included the entire implementation of the time-based JAR switch along with many unrelated changes bundled into a big PR) that this seems like the best path to me. Not ideal, but I think it's more demonstrative of the work involved in an upgrade than just gesturing at the whole PR.